openraft/src/progress/bench/vec_progress_update.rs

```diff
@@ -11,7 +11,7 @@ use crate::quorum::Joint;
 fn progress_update_01234_567(b: &mut Bencher) {
     let membership: Vec<Vec<u64>> = vec![vec![0, 1, 2, 3, 4], vec![5, 6, 7]];
     let quorum_set = Joint::from(membership);
-    let mut progress = VecProgress::<u64, u64, _>::new(quorum_set);
+    let mut progress = VecProgress::<u64, u64, _>::new(quorum_set, 0..=7);
```

Comment:
I don't know what this is about but I have to make the benchmark compile or I can test nothing :(

Reply:
I don't get what this modification is about. 🤔

Reply:
> I don't know what this is about but I have to make the benchmark compile or I can test nothing :(

Oh, I see. My mistake. Let me fix this in another PR.

Reply:
Right. Thanks. My mistake!

Reply:
It is a micro-benchmark for `VecProgress`, which is used for the leader to track replication states to every follower/learner.